src/main/java/org/symphonyoss/symphony/messageml/elements/TableSelect.java

```diff
@@ -0,0 +1,178 @@
+/*
+ * Copyright 2016-2017 MessageML - Symphony LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.symphonyoss.symphony.messageml.elements;
+
+import org.commonmark.node.Node;
+import org.symphonyoss.symphony.messageml.exceptions.InvalidInputException;
+import org.symphonyoss.symphony.messageml.markdown.nodes.form.TableSelectNode;
+import org.symphonyoss.symphony.messageml.util.XmlPrintStream;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * Class representing a TableSelect inside a Form.
+ *
+ * @author Pedro Sanchez
+ * @since 05/30/2019
+ */
+public class TableSelect extends FormElement {
```

Comment:
How are you going to parse from PresentationML (which is an entirely valid form of input) to the correct element type? I don't see any code or any tests for it.